angularjs.test.js

```diff
@@ -0,0 +1,40 @@
+import 'jest-dom/extend-expect'
+import {getQueriesForElement, fireEvent} from 'dom-testing-library'
+import angular from 'angular'
+
+angular.module('myApp', [])
+  .component('myCounter', {
+    template: `
+      <button ng-click="$ctrl.increment()">{{$ctrl.count}}</button>
+    `,
+    controller: function MyCounter() {
+      this.count = 0;
+      this.increment = function () {
+        this.count += 1;
+      }
+    }
+  })
+
+function render(ui) {
+  const container = document.createElement('div')
+  container.setAttribute('ng-app', 'myApp')
```

Comment:
If my old angular knowledge serves me well, this line is unnecessary because we're calling `angular.bootstrap` below.